native/src/modules/theme/hocs/withTheme.js

```diff
@@ -5,7 +5,7 @@ import type { ThemeType } from '../constants'
 import { lightTheme } from '../constants'
 import wrapDisplayName from '../../common/hocs/wrapDisplayName'
 
-function withTheme<Props: { theme: ThemeType }> (
+function withTheme<Props: { theme: ThemeType, ... }> (
```

Comment:
why is that change?

Reply:
This indicates to flow that the object contains more properties without having to specify all props (which is not possible here anyway). It makes warnings go away :grin: